executor/src/wasm_executor.rs

```diff
@@ -89,21 +91,39 @@ impl ReadPrimitive<u32> for MemoryInstance {
 	}
 }
 
+fn ascii_format(asciish: &[u8]) -> String {
+	let mut r = String::new();
+	let mut latch = false;
+	for c in asciish {
+		match (latch, *c) {
+			(false, 32...127) => r.push(*c as char),
+			_ => {
+				if !latch {
+					r.push('#');
+					latch = true;
+				}
+				r.push_str(&format!("{:02x}", *c));
+			}
+		}
+	}
+	r
+}
+
 impl_function_executor!(this: FunctionExecutor<'e, E>,
 	ext_print_utf8(utf8_data: *const u8, utf8_len: u32) => {
 		if let Ok(utf8) = this.memory.get(utf8_data, utf8_len as usize) {
 			if let Ok(message) = String::from_utf8(utf8) {
-				info!(target: "runtime", "{}", message);
+				println!("{}", message);
```

Comment:
Any good reason to revert to `println!`?

Reply:
logging doesn't work in tests, which is the primary motivation for having this.